app/containers/App/Header/WalletBlockHeight/index.js

```diff
@@ -1,20 +1 @@
-// @flow
-import { compose } from 'recompose'
-import { withCall, withData, withRecall } from 'spunky'
-
-import WalletBlockHeight from './WalletBlockHeight'
-import withNetworkData from '../../../../hocs/withNetworkData'
-import blockHeightActions from '../../../../actions/blockHeightActions'
-
-const mapBlockHeightDataToProps = (blockHeight: ?number): {
-  blockHeight: ?number
-} => ({
-  blockHeight
-})
-
-export default compose(
-  withNetworkData(),
-  withCall(blockHeightActions),
-  withData(blockHeightActions, mapBlockHeightDataToProps),
-  withRecall(blockHeightActions, ['networkId'])
-)(WalletBlockHeight)
+export { default } from './WalletBlockHeight'
```

Comment:
Instead of getting the data from `blockHeightActions` you just need to get it from `balanceActions` (since balanceActions now combines block height + balances)

Reply:
Is there a way to get that data without actually invoking the action twice?

Reply:
Try `withData(balancesActions, mapBalanceDataToProps)`